src/Forms/samples/Maui.Controls.Sample.Droid/MainApplication.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Android.App;
+using Android.Runtime;
+using Xamarin.Platform;
+
+namespace Maui.Controls.Sample.Droid
+{
+	[Application]
+	public class MainApplication : MauiApplication<MyApp>
+	{
+		public MainApplication(IntPtr handle, JniHandleOwnership ownerShip) : base(handle, ownerShip)
```

Comment:
@dellis1972 it looks like they are planning on using this for default Maui apps

Is there still some issue with Fast Deployment and a custom application class?

Reply:
There is but only for Enhanced Fast Deployment. That is where we fast deploy the dex files. It just won't work since we need the app class  (java) to be in the classes.dex in the apk. Which it won't be. There is a possible way around it but it would involve allot of work.